en/docs/install-and-setup/setup/reference/common-runtime-and-configuration-artifacts.md

```diff
@@ -52,7 +53,7 @@ The following are the artifacts used commonly in a WSO2 API Manager and API Mana
 
 ### Persistent Runtime Artifacts
 
-1. `<API-M_ANALYTICS_HOME>/repository/deployment/server` - Contains webapps, execution plans, event receivers, etc that are related to customizing WSO2 APIM Analytics during a deployment. Required for deploying a super tenant.
+1. `<API-M_ANALYTICS_HOME>/repository/deployment/server` - Contains webapps, execution plans, event receivers, etc that are related to customizing WSO2 APIM Analytics during a deployment. Required for deploying a super tenant. It is sufficient to only share the synapse configurations and execution plans between the nodes.
```

Comment:
Always use the proper capitalization etc. for industry names.
"synapse" should be "Synapse"